src/main/java/org/cryptomator/common/vaults/VaultListManager.java

```diff
@@ -101,6 +118,7 @@ private VaultSettings newVaultSettings(Path path) {
 	private void addAll(Collection<VaultSettings> vaultSettings) {
 		Collection<Vault> vaults = vaultSettings.stream().map(this::create).toList();
 		vaultList.addAll(vaults);
+		vaults.forEach(this::setVaultScheme);
```

Comment:
All other vault settings are taken from settings file, so we should not make an exception for the vault scheme. If a migration is needed, we should do it where we need the config anyway (i.e. during unlock in the unlock workflow)